uzsu/__init__.py

```diff
@@ -46,52 +46,98 @@
 
 
 import logging
+from lib.model.smartplugin import SmartPlugin
+
 from datetime import datetime, timedelta
 
 from dateutil.rrule import rrulestr
 from dateutil import parser
 
 import lib.orb
 
-class UZSU():
+class UZSU(SmartPlugin):
+    """
+    Main class of the UZSU Plugin. Does all plugin specific stuff and provides
+    the update functions for the items
+    """
+    
+    ALLOW_MULTIINSTANCE = False
+    
+    PLUGIN_VERSION = "1.3.0"
+
 
     _items = {}         # item buffer for all uzsu enabled items
 
-    def __init__(self, smarthome, path=None):
+    def __init__(self, smarthome, path=None, *args, **kwargs):
+        """
+        Initalizes the plugin. The parameters describe for this method are pulled from the entry in plugin.conf.
+
+        :param sh:  The instance of the smarthome object, save it for later references
```

Comment:
Additionally the documentation for `path` parameter can be added, too. Just to make it complete.